apis/nucleus/src/components/listbox/hooks/useOnTheFlyModel.jsx

```diff
@@ -61,6 +62,7 @@ export default function useOnTheFlyModel({ app, fieldIdentifier, stateName, opti
   const listdef = {
     qInfo: {
       qType: 'njsListbox',
+      qId: uid(),
```

Comment:
I think this is a bad ide
and that it will casue that useSessionModel will create a new model all the time

maybe use

```javascript
const [qId] = useState(() => uid());
```

or

```javascript
const id = useRef();
if (!id.current) { id.current = uid(); }
```

to get a more stable random id

Reply:
agree, that is a bit stupid. It is also strange that the ListBoxWrapper is a React component at all as the input properties never change. If you mount again we create a new one, so all that hooking stuff with useBlaBlaModel could be vastly simplified. There seesm to be a tendency to write more hooks than actually needed.